Makefile

```diff
@@ -9,7 +9,7 @@ TEMPDIR := $(shell mktemp -d)
 
 all: install
 install: $(INSTALL_STAMP)
```

Comment:
in my other commit, I propose rather to add virtualenv (the target) here instead. Do you know which one is better?